using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -1,150 +1,261 @@
 # Adobe Campaign Standard API reference
 
-## Version of the Campaign Standard extension
+---
 
-The `extensionVersion()` API returns the version of the Campaign Standard extension that is registered with the Mobile Core extension.
+### extensionVersion
 
-To get the version of the Campaign Standard extension, use the following code sample:
+Returns the running version of the Campaign extension.
```

Comment:
Sorry for the double comment on this, but based on our guidelines this should be kept as `the Campaign Standard extension`